tests/driver/screenshot/main.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+#![deny(warnings)]
+
+#[cfg(test)]
+include!(concat!(env!("OUT_DIR"), "/generated.rs"));
+
+fn main() {
+    println!("Nothing to see here, please run me through cargo test :)");
```

Comment:
We should somehow have a way to generate the images and compare them

Reply:
done